run_checkpoints.sh

```diff
@@ -20,4 +20,7 @@ for c in $INPUT_CHECKPOINTS;do
     great_expectations checkpoint run $c
 done
 
+# Build the ephemeral docs site
+build_gh_action_site.py
```

Comment:
```suggestion
./build_gh_action_site.py
```

Adding appropriate shell command to execute python file